collect_app/src/main/java/org/odk/collect/android/widgets/ListMultiWidget.java

```diff
@@ -68,11 +68,11 @@
 @SuppressLint("ViewConstructor")
 public class ListMultiWidget extends QuestionWidget implements MultiChoiceWidget {
 
-    private boolean checkboxInit = true;
+    private static final boolean checkboxInit = true;
```

Comment:
`CONSTANT_CASE` (use refactor > rename)

Reply:
Maybe this field doesn’t need to exist at all, if it’s always true.

Reply:
Oh wow, you are very right. I think this class may have been copied from `SelectMultiWidget` and it's an artifact from there. Agreed it can be removed.